### PR TITLE
docs: add mention of `@eslint-react/eslint-plugin`

### LINKED ISSUE #19265: Docs: add mention of alternative `@eslint-react/eslint-plugin` in important note for React in `Specifying Parser Options`
### Docs page(s)

https://eslint.org/docs/latest/use/configure/language-options#specifying-parser-options:~:text=We%20recommend%20using%20eslint%2Dplugin%2Dreact%20if%20you%20are%20using%20React.

### What documentation issue do you want to solve?

A new ESLint plugin for React has rules from `eslint-react-plugin`, but adds more. Also, it's more aligned with recommendations from React docs: https://eslint-react.xyz/docs/faq

### What do you think is the correct solution?

Add mention of `@eslint-react/eslint-plugin` next to `eslint-plugin-react` like:

"We recommend using [eslint-plugin-react](https://github.com/jsx-eslint/eslint-plugin-react) or [@eslint-react/eslint-plugin](https://github.com/rel1cx/eslint-react) if you are using React."

### Participation

- [x] I am willing to submit a pull request for this change.

### Additional comments

_No response_

### PR DESCRIPTION
**Pull request by rakleed**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)

Closes #19265

### CODE REVIEW

**Review by amareshsm — DISMISSED** [review]
LGTM, thanks!

Would like another team member to review this.

File: docs/src/use/configure/language-options.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ export default [
 ```
 
 ::: important
-Please note that supporting JSX syntax is not the same as supporting React. React applies specific semantics to JSX syntax that ESLint doesn't recognize. We recommend using [eslint-plugin-react](https://github.com/jsx-eslint/eslint-plugin-react) if you are using React.
+Please note that supporting JSX syntax is not the same as supporting React. React applies specific semantics to JSX syntax that ESLint doesn't recognize. We recommend using [`@eslint-react/eslint-plugin`](https://github.com/rel1cx/eslint-react) or [`eslint-plugin-react`](https://github.com/jsx-eslint/eslint-plugin-react) if you are using React.
```
Comment (aladdin-add) [inline docs/src/use/configure/language-options.md:68]: I don't consider `eslint-plugin-react` to be the 2nd choice - it's still more popular and is still maintained.

**Review by aladdin-add — CHANGES_REQUESTED** [review]

File: docs/src/use/configure/language-options.md (hunk 1 above)
Comment (rakleed) [reply docs/src/use/configure/language-options.md]: 7b4c05b

**Review by aladdin-add — APPROVED** [review]
LGTM. Would like another review before merging.

**Comment by nzakas** [comment]
Sorry, `@eslint-react/eslint-plugin` isn't popular enough to mention alongside `eslint-plugin-react`.

**Comment by rakleed** [comment]
@nzakas

> `@eslint-react/eslint-plugin` isn't popular enough

What are the exact criteria? How many monthly downloads on NPM or how many stars on GitHub?

**Comment by nzakas** [comment]
Once you get above 1 million downloads per week, come back and let us know.

**Comment by rakleed** [comment]
@nzakas it now has more than one million [downloads](https://www.npmjs.com/package/@eslint-react/eslint-plugin) per week:

CleanShot 2026-05-01 at 00 33 24

Could you please accept this PR?

**Pull request closed without merging**